src/main/scala/com/bwsw/kafka/reader/Consumer.scala

```diff
@@ -60,69 +60,90 @@ class Consumer[K,V](settings: Consumer.Settings) {
     * @param topicPartitionInfoList entity which contains topics, partitions and offsets.
     */
   def assignWithOffsets(topicPartitionInfoList: TopicPartitionInfoList): Unit = {
+    logger.trace(s"assignWithOffsets(topicPartitionInfoList: $topicPartitionInfoList)")
     val topicPartitionsWithOffsets = topicPartitionInfoList.entities.map {
       case TopicPartitionInfo(topic, partition, offset) =>
         (new TopicPartition(topic, partition), offset)
     }
-    consumer.assign(
-      topicPartitionsWithOffsets.map {
-        case (topicPartitions, offset) => topicPartitions
-      }.asJavaCollection
-    )
+
+    val topicPartitions = topicPartitionsWithOffsets.map {
+      case (partition, offset) => partition
+    }
+
+    logger.debug(s"assign topic partitions: $topicPartitions")
+    consumer.assign(topicPartitions.asJavaCollection)
+
+    logger.debug(s"seek topic partitions with offsets: $topicPartitionsWithOffsets")
     topicPartitionsWithOffsets.foreach {
-      case (topicPartition, offset) => consumer.seek(topicPartition, offset)
+      case (partition, offset) => consumer.seek(partition, offset)
     }
   }
 
   /**
     * Assign a list of topic/partition to this consumer
     *
     * @param topicInfoList entity which contains topics
-    * @throws NoSuchElementException if no one of the topics does not exist in Kafka
+    * @throws NoSuchElementException if no one of topics exists in Kafka
     */
   def assign(topicInfoList: TopicInfoList): Unit = {
-    val topicPartitions = covertToTopicPartition(topicInfoList)
+    logger.trace(s"topicInfoList: $topicInfoList")
+    val topicPartitions = convertToTopicPartition(topicInfoList)
     if (topicPartitions.nonEmpty) {
+      logger.debug(s"assign topic partitions: $topicPartitions")
       consumer.assign(topicPartitions.asJavaCollection)
     } else {
-      throw new NoSuchElementException(s"No one of topics: $topicInfoList does not exist")
+      logger.error(s"No one of topics: $topicInfoList exists, NoSuchElementException will be thrown")
```

Comment:
you should define the uniform format of logs: the uppercase or the lowercase first letter

Reply:
Uppercase, excluding messages for "trace" log level